server/tables/tables.go

```diff
@@ -145,7 +145,7 @@ type Group struct {
 	// An unique URL segment that is displayed in group-related URLs.
 	// e.g. "example-org" in app.buildbuddy.com/join/example-org or
 	// "example-org.buildbuddy.com" if we support subdomains in the future.
-	URLIdentifier string
+	URLIdentifier string `gorm:"unique"`
```

Comment:
Maybe this should be unique_index so we can do lookups on it later?

https://stackoverflow.com/questions/53611325/what-is-the-difference-between-unique-index-and-unique

Reply:
Done